docs/release-cycle.md

```diff
@@ -17,65 +16,62 @@ That said, nothing here is set in stone. See this as inspiration :)
- - Go through _every_ open issue in the kubeadm repo.
-   - As there are a small number of core maintainers working on kubeadm, there often isn't time to answer all feature requests or bugfixes, or to close old, outdated ones.
-   - As a general practice, every issue must be triaged **at least** once a quarter.
- - Write proposals for new and larger features
-   - Often starting as a Google Doc that is easy to comment on/let others collaborate on.
-   - Get consensus on the design in the SIG
-   - Convert to markdown, send PR to `kubernetes/community` and loop in the rest of the community (other relevant SIGs, if not looped in already)
-   - Update as needed by review comments, get LGTM and merge.
- - Start coding new features, fix bugs and increase unit testing coverage levels
- - Ensure that the [`kubeadm-gce`](https://k8s-testgrid.appspot.com/sig-cluster-lifecycle#kubeadm-gce) is master blocking.
-   - "Master blocking" means that this is the dashboard the release team looks at when they decide whether to cut an alpha.X release or not.
- - Cleanup work. As kubeadm supports deploying clusters one minor version below the CLI version, support for v1.7-specific workarounds may be dropped when the v1.9 cycle starts
-   - A good example of this is Bootstrap Tokens and RBAC rules.
-   - In v1.7, alpha Bootstrap Tokens were used. On upgrade to v1.8 they were slightly modified to use beta features and resource names
-   - In v1.7, the `extensions/v1beta1` and `rbac.authorization.k8s.io/v1beta1` groups were used. In v1.8, `apps/v1beta2` and `rbac.authorization.k8s.io/v1` were added. Early in the v1.9 we can switch over to use `apps/v1beta2` and `rbac.authorization.k8s.io/v1` as those groups will always exist for the supported versions.
-   - These are good examples of cleanup work that can be done. Also, `cmd/kubeadm/app/constants.MinimumControlPlaneVersion` should be bumped (in this example from `v1.7.0` to `v1.8.0`)
-   - You might want to wait some time from the previous .0 release to make it easier to cherrypick possible bug fixes.
+- Create a new housekeeping tracking issue similar to [this one](https://github.com/kubernetes/kubeadm/issues/180).
+- Remove e2e tests for k8s/kubeadm versions outside of the support skew.
+- Manage cherrypicks for the current stable branch
+  - Make sure high-priority bug reports are triaged and fixed.
+  - Create bug fixes, ensure they're merged, and cherry pick them to the release branch.
+  - Coordinate with the patch release team.
+- Plan/prioritize features, testing work and bugs
+  - At the beginning of each cycle the kubeadm team has a planning session during the kubeadm office hours meeting.
+  - There people sign up for working on the high level features and objectives for the release.
+  - Based on the planning session shift issues between milestones in the `kubernetes/kubeadm` repository.
+  Create new issues if needed.
```

Comment:
IMO the issue part goes after the KEP part; might be worth to add something on the lines of.
- break down the implementation of larger feature into smaller actionable issue, so the work can be federated
- when possible, create help want/good for first contributor issues
- use a tracking issue to track/coordinate all the efforts